app/src/main/AndroidManifest.xml

```diff
@@ -331,10 +331,28 @@
             android:name="com.duckduckgo.app.email.ui.EmailWebViewActivity"
             android:label="@string/emailProtectionWebViewActivityTitle"
             android:parentActivityName="com.duckduckgo.app.email.ui.EmailProtectionActivity" />
+
+        <activity
+            android:name="com.duckduckgo.app.WidgetThemeConfiguration"
+            android:label="@string/widgetConfigurationActivityTitle"
+            android:theme="@style/AppTheme.Transparent">
+            <intent-filter>
+                <action android:name="android.appwidget.action.APPWIDGET_CONFIGURE" />
+            </intent-filter>
+        </activity>
+
         <service
             android:name="com.duckduckgo.app.job.AppConfigurationJobService"
             android:permission="android.permission.BIND_JOB_SERVICE" />
 
+        <service android:name="com.duckduckgo.widget.FavoritesWidgetService"
```

Comment:
These services are related to the gridview adapter we use in the favorites widget.